### DISCUSSION TITLE
Mac App - Integration

### DISCUSSION THREAD

**Discussion by andrei4002** [ExistentialAudio/BlackHole#658] [Q&A]
Hey, maybe this is a silly question, but is there a way BlackHole can be integrated somehow into a mac app (in the store) ?

From what I understand BlackHole works more like a separate thing, and I could create an app that would just use the devices BlackHole creates, installing it separately and such, but I'm wondering if a better integration can be implemented/distributed

**Comment by devinroth** [comment]
Apple doesn’t allow drivers in the Mac app store.

I sell non-GPL3.0 licenses for BlackHole that would allow you to integrate it into your app. Email me at ***@***.*** for specifics if you’re interested.

Devin
